Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -50,26 +60,53 @@ class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
     // MARK: - UI Setup
     func setupView() {
         contentView.backgroundColor = .clear
-        contentView.addSubview(logoImage)
-        contentView.addSubview(logoTextImage)
+        containerView.addSubview(logoImage)
+        containerView.addSubview(logoTextImage)
+        contentView.addSubview(containerView)
+
+        // TODO: Felt Privacy - Private mode in Redux to follow
+        let isiPadAndPrivate = UIDevice.current.userInterfaceIdiom == .pad && false
+        let logoSizeConstant = isiPadAndPrivate ? UX.Logo.iPadImageSize : UX.Logo.iPhoneImageSize
```

Comment:
nit - I like how there is a separate boolean to make the code more readable. What do you think about doing the same for the other constraints (i.e. top constant, width, leading constant) for consistency?